app/assets/stylesheets/structure.scss

```diff
@@ -1,10 +1,55 @@
+*, *:before, *:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
 .container {
   position: relative;
+  margin: 0 auto;
+  width: 90%;
 }
 
-@media screen and (min-width: 1000px) {
+.flex-grid {
+  display: block;
+  .col {
+    width: 100%;
+    margin-bottom: 1em;
+  }
+}
+
+section { margin-bottom: 4em; }
+
+
+@media screen and (min-width: 800px) {
   .container {
-    margin: 0 auto;
     max-width: 1020px;
   }
+
+  .flex-grid {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
```

Comment:
You can use [Compass' display-flex](http://compass-style.org/reference/compass/css3/flexbox/#mixin-display-flex) to combine lines 30-32.